run.py

```diff
@@ -1,6 +1,10 @@
 from flask import Flask
 from config import get_env_config
 
+from dotenv import load_dotenv
```

Comment:
can you please remove this file change since this is part of another potential issue

Reply:
sure, done